scripts/lint-staged/eslint.js

```diff
@@ -31,7 +31,13 @@ function groupFilesByPackage() {
   );
 
   for (const file of files) {
-    const packagePath = packagesWithEslint.find(packagePath => file.startsWith(packagePath));
+    const packagePath = packagesWithEslint.find(packagePath => {
+      // if file lives within searched package we will get only shortened absolute path `/src/abc.ts`
+      // we add `.` to make it relative and thus have match pattern to check upon
+      const normalizedFilePath = file.replace(packagePath, '.');
```

Comment:
this is kinda naive but works. otherwise we would need to do real checks against files system which would be not the most performant solution. 

If anyone has better idea how to handle please share !